pkg/generate/app/pipeline.go

```diff
@@ -140,24 +141,24 @@ func (g PipelineGroup) String() string {
 	return strings.Join(s, "+")
 }
 
-const MaxServiceNameLen = 24
+const maxServiceNameLength = 24
```

Comment:
It seems like we need to set maxServiceNameLength to 19 so the name generator doesn't make it > 24 chars.